eox_nelp/stats/decorators.py

```diff
@@ -0,0 +1,37 @@
+"""eox-nelp Metrics file.
+
+decorators:
+    cache_method: Cache the result of the inner method.
+"""
+from django.conf import settings
+from django.core.cache import cache
+
+
+def cache_method(func):
+    """
+    Cache the function result to improve the response time.
+
+    Args:
+        func<function>: Target function to be cached.
+
+    Return:
+        <funtion>: Wrapper function.
+    """
+    def wrapper(*args, **kwargs):
+        key = f"{func.__name__}.{'-'.join(map(str, args))}.STATS_CACHE_KEY"
```

Comment:
I think that is better to relate more to the tenant the cache key, it could be other stats. `STATS_CACHE_KEY`  to `TENANT_STATS_CACHE_KEY`

Reply:
my question would be what are you talking about tenants here ? I mean we are in a multitenant context, however here we are talking about  a single method which functionality is just to cache the inner function result and that method shouldn't be aware of multi-tenant instances or something similar, therefore if we use   `TENANT_STATS_CACHE_KEY` that makes me think that the method has different caches from tenant  and that is false